.gitignore

```diff
@@ -85,7 +85,7 @@ out
 
 # Nuxt.js build / generate output
 .nuxt
-dist
+# dist
```

Comment:
maybe keep dist out of the picture?

Reply:
Yep, needed to make the branch in BE compile before releasing the type

Reply:
but you can revert it now right?

Reply:
Yessir

Reply:
before merge tho?